src/AdvicesContainer.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from "react";
+import "./index.css";
+import Button from "./Button";
+import Advice from "./Advice";
+import pauseMb from "./images/pattern-divider-mobile.svg";
+import pauseDt from "./images/pattern-divider-desktop.svg";
+const url = "https://api.adviceslip.com/advice";
+
+const AdvicesContainer = () => {
+	const [advices, setAdvices] = useState("");
+	const [loading, setLoading] = useState(true);
+
+	const handleClick = async () => {
+		setLoading(true);
+
+		const response = await fetch(url);
+
+		const result = await response.json();
+
+		console.log("slip is: ", JSON.stringify(result.slip, null, 4));
+
+		setAdvices(result.slip);
+	};
+
+	useEffect(() => {
+		if (loading) {
+			setTimeout(() => {
+				setLoading(false);
+			}, 1000);
+		}
+	}, [loading]);
```

Comment:
This sets loading to false not when data is fetched, but just automatically one second after the loading state has changed. Does it make sense to do it like that? What if the data is not fetched after the 1 second? 🤓 
I would highly recommend to look into a more standard fetching and loading state management, there are plenty of examples out there 💡